i18n/en-US/core/help.json

```diff
@@ -0,0 +1,26 @@
+{
+    "DESCRIPTION": "Get general information about TypicalBot or help with a specific command.",
+    "USAGE": "help [command]",
+    "NONE": "\u200B\t\tHello there, I'm TypicalBot! I was created by HyperCoder#2975 and nsylke#4490. If you would like to access my list of commands, try using `{{prefix}}commands`. If you need any help with commands or settings, you can find documentation at **<{{docs}}>**. If you cannot figure out how to use a command or setting, or would like to chat, you can join us in the TypicalBot Lounge at **<{{server}}>**.\n\n\t\t*Built upon over a year of experience, TypicalBot is the ironically-named bot that is far from typical. Stable, lightning fast, and easy to use, TypicalBot is there for you and will seamlessly help you moderate your server and offer some entertaining features for your users, every step of the way.*",
+    "TYPICAL_INFO": "TypicalBot Info",
+    "INVALID": "There is not a command named `{{name}}`. Try searching something else.",
+    "INVALID_INFO": "Invalid Command Input",
+    "TEXT": "$t(core/help:TEXT_1)\n$t(core/help:TEXT_2)\n$t(core/help:TEXT_3)\n\n$t(core/help:TEXT_4)\n$t(core/help:TEXT_5)\n$t(core/help:TEXT_6)\n$t(core/help:TEXT_7)\n$t(core/help:TEXT_8)\n$t(core/help:TEXT_9)\n$t(core/help:TEXT_10)",
+    "TEXT_1": "**__Usage For:__** {{name}}",
+    "TEXT_2": "**[Param]** means a parameter is optional.",
+    "TEXT_3": "**<Param>** means a parameter is required.",
+    "TEXT_4": "```",
+    "TEXT_5": "Command: {{commandName}}",
+    "TEXT_6": "Aliases: {{aliases}}",
+    "TEXT_7": "Permission: {{permission}}",
+    "TEXT_8": "Description: {{description}}",
+    "TEXT_9": "Usage: {{usage}}",
+		"TEXT_10": "```",
+		"COMMAND_USAGE": "Command Usage: {{name}}",
+		"PARAMETERS": "• [[Parameter]]() - Optional Parameter\n• [<Parameter>]() - Required Parameter",
+		"COMMAND": "» Command",
+		"ALIASES": "» Aliases",
+		"PERMISSION": "» Permission",
+		"DESC": "» Description",
+		"USE": "» Usage"
```

Comment:
After uploading the strings to Crowdin, these special characters will not render properly and/or become other characters. Can we use unicode instead? The unicode for `»` is `\u00bb`.